test/mock/MockERC20.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.25 <0.9.0;
+
+// solhint-disable custom-errors,max-line-length
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract MockERC20 is ERC20 {
```

Comment:
Tried to used openzeppelin mock contracts but they are not included in the dependencies. I have noticed that usually other projects follow this strategy, so just moved the mock contracts to a mock folder inside the testing one.

Reply:
Use the pre-existing `TestERC20.sol` file please. That file contains multiple mock ERC20s